modules/fieldlinessequence/rendering/renderablefieldlinessequence.h

```diff
@@ -35,12 +35,31 @@
 #include <openspace/properties/vector/vec2property.h>
 #include <openspace/properties/vector/vec4property.h>
 #include <openspace/rendering/transferfunction.h>
-#include <atomic>
+#include <openspace/util/dynamicfilesequencedownloader.h>
+#include <vector>
 
 namespace openspace {
 
 class RenderableFieldlinesSequence : public Renderable {
 public:
+    enum class LoadingType {
+        StaticLoading,
+        DynamicLoading,
+        DynamicDownloading
```

Comment:
What is the difference between dynamic loading and downloading?  Is this documented anywhere?